app.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const bodyParser = require("body-parser");
+const port = 3000;
+const School = require("./School");
+const Student = require("./Student");
+
+app.use(cors());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+let mySchool = new School();
+
+// Create a Class w/ Course Name and Teacher
+app.post("/class", (req, res) => {
+    try {
+        let className = req.body.name;
+        let classTeacher = req.body.teacher;
+        mySchool.addClass(className, classTeacher);
+        res.status(200).json({
+            class: {className,classTeacher,students: []},
```

Comment:
Include your new variable `newClass` as a value to a key `class`. Otherwise here you are returning your request back as a response - creating an illusion that everything went well. Also you are hardcoding `students` here